SPECS/kernel/config

```diff
@@ -1008,7 +1008,7 @@ CONFIG_COMPAT_BINFMT_ELF=y
 CONFIG_ELFCORE=y
 CONFIG_CORE_DUMP_DEFAULT_ELF_HEADERS=y
 CONFIG_BINFMT_SCRIPT=y
-CONFIG_BINFMT_MISC=y
+CONFIG_BINFMT_MISC=m
```

Comment:
Could you copy these changes to aarch64 to keep the configs aligned?

Reply:
Done. For all the config options under consideration, I have made sure the same value is reflected in aarch64. I have also add ARM64 for all the config options too. Please let me know if that is what you intended.